packages/ui/stories/Button.stories.ts

```diff
@@ -1,44 +1,75 @@
-import { Meta, Story } from '@storybook/vue3'
-import { Component, FunctionalComponent, h, Slot } from 'vue'
+import { defineMeta, defineStory } from './util'
 import { SButton } from '@/lib'
+import { IconClose } from '@/components/icons'
 
-const meta: Meta = {
-  title: 'Example/Button',
-}
-
-export default meta
-
-interface Props {
-  type?: 'primary' | 'secondary'
-  rounded?: boolean
-}
-
-interface Args {
-  props?: Props
-  defaultSlot: string | (() => Slot)
-}
+export default defineMeta({
+  title: 'Example/Button'
+})
 
-const Template: Story<Args> = ({ props, defaultSlot }) => ({
+export const Configurable = defineStory((args) =>  ({
+  components: {
+    SButton,
+    IconClose,
+  },
   setup() {
-    return () => h(SButton, props, { default: typeof defaultSlot === 'string' ? () => defaultSlot : defaultSlot })
+    return {
+      args
+    }
   },
-})
-
-export const Primary = Template.bind({})
-Primary.args = { defaultSlot: 'Primary' }
-
-export const Secondary = Template.bind({})
-Secondary.args = {
-  props: { type: 'secondary' },
-  defaultSlot: 'Secondary',
+  template: `
+    <div class="flex flex-col items-start">
+      <SButton v-bind="args">
+        <template #icon>
+          <!--viewBox for svg scaling-->
+          <IconClose
+            viewBox="0 0 24 24"
+            style="width: 1em; height: 1em;"
```

Comment:
Isn't viewBox & weight/height applied by default thanks to `vite-svg-loader`?

Reply:
As far as I understood it removes viewBox

Reply:
Disabled this behaviour